README.md

```diff
@@ -14,6 +14,9 @@ You can also easily implement a generic *Leader Election* on top of it (see the
 
 As of now, `libkv` offers support for `Consul`, `Etcd` and `Zookeeper`.
 
+This particular fork adds TLS Client support for the etcd backend.
```

Comment:
Is this possible to remove the reference to the **fork** and instead create a new sub-section called *Security* or *TLS* in which we say that etcd has been tested and that the support for Consul and zookeeper is coming along?